recipe/meta.yaml

```diff
@@ -0,0 +1,61 @@
+{% set name = "imbalanced-ensemble" %}
+{% set version = "0.2.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  - path: requirements.txt
```

Comment:
Great solution!

Reply:
what is the point of including this file?

Reply:
alternatively this can be fetched by defining a second source:
` - git_url: https://github.com/ZhiningLiu1998/imbalanced-ensemble
   - git_rev: 0.2.1`

Reply:
the point of the file is because upstream doesn't provide a `requirements.txt`. 

I didn't use the Github repo due to it having spoty release history.

Reply:
A comment for our future selves will clarify why the file is here

Reply:
so you took the requirements from PyPi and put them in the `requirements.txt `file?

Reply:
Taken from here: https://github.com/ZhiningLiu1998/imbalanced-ensemble/blob/62dc2c7fa0c66ed2a6ee4c082a7f46f84b4cfeca/requirements.txt

Reply:
Oh, for some reason I thought they requested a less than latest version. That's ok then.
I would rather take it with the:
```
source:
  - git_url: https://github.com/ZhiningLiu1998/imbalanced-ensemble
     git_rev: 62dc2c7fa0c66ed2a6ee4c082a7f46f84b4cfeca
     folder: requirements-checkout
```
and then
```
build:
  script:
    - cp requirements-checkout/requirements.txt $SRC_DIR/requirements.txt
    - {{ PYTHON }} ...
```

Reply:
> This way you don't have to update the file manually upon an update.

@boldorider4 fair point, I pushed a change to source the `requirements.txt` file from upstream in raw (no git involved).